docs-source/spring/content/platform/config/_index.md

```diff
@@ -11,22 +11,393 @@ configuration in the Oracle Autonomous Database, so it easily supports labelled
 environments, as well as being accessible to a wide range of tools for managing the content.
 Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table.
 
-For example, the Spring Cloud Config client's Spring `application.yaml` configuration file could include:
```

Comment:
feels like there is a word missing here?  to introduce the link

Reply:
Rewrote the "init" section.